pom.xml

```diff
@@ -50,7 +50,7 @@
         <!-- This is first of two places for changing the version of SW360 -->
         <!-- note that 6.1.0-SNAPSHOT relates to 6.0.${patchlevel} -->
         <!-- pls see also the profile cli -->
-        <revision>13.4.${patchlevel}</revision>
+        <revision>14.0.${patchlevel}</revision>
         <patchlevel>0-SNAPSHOT</patchlevel>
```

Comment:
I guess `SNAPSHOT` should be  removed while creating a relaese.
And then added back after release is done?